### ISSUE TITLE
Error with Middleware

### ISSUE THREAD

**Issue by Azuka** [orchestral/tenanti#25]
I'm getting an InvalidArgumentException thrown when I try to use the example in the docs for a multi-database setup:

```php
    public function handle($request, Closure $next)
    {
        // Get account id
        $accountId = $request->route()->parameter('accountId');

        // Get account from id
        $account = Account::findOrFail($accountId);

        \Tenanti::driver()->asDefaultConnection($account, 'tenants_{id}');

        return $next($request);
    }
```

I get

```
InvalidArgumentException in TenantiManager.php line 52:
Default driver not implemented.
```

Am I doing this the wrong way?

**Comment by crynobone** [comment]
```php
Tenanti::driver('user')
```

The example clearly shown that you need to be specific with the driver name.

**Comment by Azuka** [comment]
I'm an idiot. Thanks for being patient.

**Comment by crynobone** [comment]
No worries, feel free to suggest rewording the documentation if you think
it need more explanation.

**Comment by arun0103** [comment]
Hello, I am a beginner in laravel. I am getting the same error as @Azuka . I think it is due to not adding  

    Tenanti::driver('user')->asDefaultConnection($user, 'tenants_{id}');

Can you guys help on this. I don't know where to put it.
